tests/frame/test_common.py

```diff
@@ -325,36 +325,29 @@ def test_join(df_raw: Any) -> None:
     compare_dicts(result_native, expected)
 
 
-@pytest.mark.parametrize(
-    ("df_raw", "expected"),
-    [
-        (
-            df_polars,
-            {"a": [1, 1, 1, 3, 3, 3, 2, 2, 2], "a_right": [1, 3, 2, 1, 3, 2, 1, 3, 2]},
-        ),
-        (
-            df_lazy,
-            {"a": [1, 1, 1, 3, 3, 3, 2, 2, 2], "a_right": [1, 3, 2, 1, 3, 2, 1, 3, 2]},
-        ),
-        (
-            df_pandas,
-            {"a": [1, 1, 1, 3, 3, 3, 2, 2, 2], "a_right": [1, 3, 2, 1, 3, 2, 1, 3, 2]},
-        ),
-        (
-            df_mpd,
-            {"a": [1, 1, 1, 3, 3, 3, 2, 2, 2], "a_right": [1, 3, 2, 1, 3, 2, 1, 3, 2]},
-        ),
-    ],
-)
-def test_cross_join(df_raw: Any, expected: dict[str, list[Any]]) -> None:
+@pytest.mark.parametrize("df_raw", [df_polars, df_lazy, df_pandas, df_mpd])
+def test_cross_join(df_raw: Any) -> None:
     df = nw.from_native(df_raw).select("a")
     result = df.join(df, how="cross")  # type: ignore[arg-type]
+
+    expected = {"a": [1, 1, 1, 3, 3, 3, 2, 2, 2], "a_right": [1, 3, 2, 1, 3, 2, 1, 3, 2]}
```

Comment:
I think it was parametrized because I wanted to include tests with None values?!